### PR TITLE
Secret id remains on the screen if passphrase inputted is incorrect

### PR DESCRIPTION
**Pull request by swongg**
- Secret id remains on the screen if passphrase inputted is incorrect #54

### CODE REVIEW

File: public/script.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ const fqdn = window.location.host;
 
 const DEFAULT_EXPIRY = 60 * 60; // one hour
 const PASSPHRASE_CHARSET =
-  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz~!@#$%^&*()?";
-const PASSPHRASE_LEN = 64;
+  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz@#$%^&?";
+const PASSPHRASE_LEN = 40;
```
Comment (swongg) [inline public/script.js]: I pulled from latest master upstream before creating the wrong-passphrase branch. For some reason, this update did not get included. Therefore, I'm adding this back ...

Comment (tamf) [reply public/script.js]: Let's not do that, instead pls re-pull master and rebase your feature branch. I suspect that when you pulled master my PR had not been merged yet?

Make sure to undo this change before rebase or else you will have merge conflict

Comment (tamf) [reply public/script.js]: Your change doesn't even depend on mine so theoretically you wouldn't care if you had my change or not. Once your PR is merged into master everything should be up to date in master

**Comment by tamf** [comment]
Sorry, had to force fix a previous commit on master. Pls rebase

File: public/script.js (hunk 1 above)
Comment (swongg) [reply public/script.js]: that makes sense. Okay, I'll do a rebase

File: public/script.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ const fqdn = window.location.host;
 
 const DEFAULT_EXPIRY = 60 * 60; // one hour
 const PASSPHRASE_CHARSET =
-  "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz@#$%^&?";
```
Comment (tamf) [inline public/script.js:8]: Yeah, I'm not sure exactly what's going on but most likely you aren't rebasing off the most up-to-date master... you may want to recreate your master

**Review by tamf — CHANGES_REQUESTED** [review]

File: public/script.js (hunk 2 above)
Comment (tamf) [reply public/script.js:8]: I would recommend recreating master and then cherry picking your changes onto a newly created branch

**Comment by swongg** [comment]
please refer to the new PR

**Pull request closed without merging**